charts/zipkin/ci/serviceMonitor-values.yaml

```diff
@@ -0,0 +1,6 @@
+---
+serviceMonitor:
+  enabled: true
+  interval: 1s
+  scrapeTimeout: 1s
```

Comment:
ps I don't know if this is normal in k8s, but if you make an invalid config, like scrapeTimeout > interval, the service monitor will be created, but just won't ever be processed. You end up having to look at prometheus-operator pod logs to figure it out. I don't know if this is a bug or a norm.. if someone thinks this is a bug, probably needs to be raised upstream as hours lost over this.